.release-it.json

```diff
@@ -13,7 +13,8 @@
     "requireCleanWorkingDir": false
   },
   "npm": {
-    "publish": true
+    "publish": true,
+    "skipChecks": true
```

Comment:
Just double checking this is intended

Reply:
Yeah - its a prereq check that causes the npm steps to fail because they happen before auth happens - this bypasses the initial pre-req check but still requires us to be properly auth'd to npm to publish

Reply:
Perfect!